pkg/controller/export/export_config_controller.go

```diff
@@ -124,7 +124,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 
 	err = r.system.UpsertConnection(ctx, config, request.Name, cfg.Data["driver"])
 	if err != nil {
-		return reconcile.Result{}, err
```

Comment:
why is this changed to requeue?

Reply:
This was changed because in updating connection there could be errors which would get resolved over time. For example, if "export path" is updated in "disk connection" we delete the files in old path and start writing violations to new path. However, it may be possible that a lock is held by reader process which prevents us from updating connection successfully, retrying makes sense in such cases.

Reply:
Per controller-runtime:
> // If the returned error is non-nil, the Result is ignored and the request will be
	// requeued using exponential backoff.

https://github.com/kubernetes-sigs/controller-runtime/blob/d8b679322d5cc466d12301b8875c42040fd29e46/pkg/reconcile/reconcile.go#L112-L114

Reply:
removed requeuing.